pegasus/sites.v3/code.org/public/accessibility.haml

```diff
@@ -0,0 +1,100 @@
+
+%section.accessibility-explore
+  .wrapper
+    %h2=hoc_s(:accessibility_explore_title)
+    .action-block.action-block--two-col
+      %img{src: "/images/accessibility/explore.jpg", alt: hoc_s(:accessibility_explore_title), class: "col-50"}
+      .text-wrapper.col-50
+        %p.heading-sm=hoc_s(:accessibility_explore_text)
+        %a{href: "https://support.code.org/hc/en-us/sections/15186591538957-Accessibility", class: "link-button", target: "_blank"}
+          =hoc_s(:accessibility_explore_button)
+    .clear
+
+%section.bg-neutral-light.accessibility-help
+  .wrapper
+    %h2=hoc_s(:accessibility_help_us_title)
+    %p=hoc_s(:accessibility_help_us_text)
+    %a{href: "https://support.code.org/hc/en-us/sections/15186591538957-Accessibility", target: "_blank", class: "link-button"}
+      =hoc_s(:accessibility_help_us_button)
+
+%section.accessibility-commit
+  .wrapper
+    %h2=hoc_s(:accessibility_commit_title)
+    - commit_index = 4
+    - commit_index.times do |index|
+      %p=hoc_s("accessibility_commit_text_#{index + 1}")
+
+= view :section_divider_line
+
+- partner_index = 10
+- accessibility_partner_links = ["https://www.washington.edu/accesscomputing/accesscsforall","https://milnel2.github.io/blocks4alliOS/","https://codejumper.com/","https://www.csforall.org/projects_and_programs/accessibility-pledge/","https://www.deafkidscode.org/","https://www.apple.com/education/docs/everyone-can-code-curriculum-guide.pdf","https://www.khanacademy.org/","https://www.aph.org/product/snap-circuits-jr-access-pack/","https://sphero.com/products/sphero-bolt","https://quorumlanguage.com/lessons/code.html"]
+- accessibility_partner_tags = [hoc_s(:module_label_student_content_resources),hoc_s(:module_label_student_product),hoc_s(:module_label_student_product),hoc_s(:module_label_student_content_resources),hoc_s(:module_label_curriculum_pure),hoc_s(:module_label_student_product),hoc_s(:module_label_curriculum_pure),hoc_s(:module_label_student_product),hoc_s(:module_label_student_product),hoc_s(:module_label_curriculum_pure)]
+- accessibility_partner_levels = [": K-12", ": K-8", ": K-6",": K-12", ": K-12", ": K-12", ": K-12", ": K-6", ": K-12", ": K 9-12"]
+
+%section.accessibility-partner
+  .wrapper
+    %h2=hoc_s(:accessibility_partner_title)
+    %p=hoc_s(:accessibility_partner_desc)
+    .action-block__wrapper.action-block__wrapper--three-col
+      - partner_index.times do |index|
+        .action-block.action-block--one-col.flex-space-between
+          .content-wrapper
+            %p.overline
+              =accessibility_partner_tags[index]
+            %h3=hoc_s("accessibility_card_title_#{index + 1}")
+            %p
+              =hoc_s(:grade_level)
+              =accessibility_partner_levels[index]
+            %p=hoc_s("accessibility_card_text_#{index + 1}")
+          .content-footer
+            %a.link-button{href: accessibility_partner_links[index], target: "_blank"}
+              = hoc_s(:call_to_action_learn_more)
```

Comment:
Can you add aria-labels to the buttons in this section to be more descriptive? Similar to what you did here on the AI PL 101 page: https://github.com/code-dot-org/code-dot-org/blob/8996b0805e8c1b8356736f71cfa33ca0f7db8977/pegasus/sites.v3/code.org/public/ai/pl/101/index.haml#L61

Reply:
Thanks for adding these! Can you update the text though so the button action is more descriptive, like `Learn more about [accessibility_card_title…]`.